html/global_attributes.json

```diff
@@ -130,9 +130,16 @@
                 "notes": "Originally only supported on the <code>&lt;input&gt;</code> element."
               }
             ],
-            "edge": {
-              "version_added": null
-            },
+            "edge": [
+              {
+                "version_added": "≤79",
+                "notes": "Support added for <code>autocomplete</code> on the <code>&lt;textarea&gt;</code> and <code>&lt;select&gt;</code> elements."
+              },
+              {
+                "version_added": "≤79",
+                "notes": "Originally only supported on the <code>&lt;input&gt;</code> element."
```

Comment:
I think these notes and therefore the range doesn't apply to Edge. Can be just version_added: 79. Do you think so, too?